src/libraries/adaptive-curve/ConstantsLib.sol

```diff
@@ -14,12 +19,12 @@ library ConstantsLib {
     /// @notice Target utilization (scaled by WAD).
     int256 public constant TARGET_UTILIZATION = 0.9 ether;
 
-    /// @notice Initial rate at target per second (scaled by WAD).
-    int256 public constant INITIAL_RATE_AT_TARGET = int256(0.04 ether) / 365 days;
+    /// @notice Initial rate at target per second (scaled by WAD) (1% min APR / 16% max APR).
+    int256 public constant INITIAL_RATE_AT_TARGET = 4_00 * ONE_BPS / 365 days;
 
-    /// @notice Mininimum rate at target per second (scaled by WAD) (0.1% APR).
-    int256 public constant MIN_RATE_AT_TARGET = int256(0.001 ether) / 365 days;
+    /// @notice Minimum rate at target per second (scaled by WAD) (0.025% min APR).
+    int256 public constant MIN_RATE_AT_TARGET = 10 * ONE_BPS / 365 days;
```

Comment:
Same as comment on max rate at target, but for min rate (min rate at target is 0.1% APR)